x-pack/plugins/ml/server/models/annotation_service/annotation.ts

```diff
@@ -332,6 +342,79 @@ export function annotationProvider({ asInternalUser }: IScopedClusterClient) {
     }
   }
 
+  /**
+   * Fetches the latest delayed data annotation per job.
+   * @param jobIds
+   */
+  async function getDelayedDataAnnotations({
+    jobIds,
+    earliestMs,
+  }: {
+    jobIds: string[];
+    earliestMs?: number;
+  }): Promise<Annotation[]> {
+    const params: estypes.SearchRequest = {
+      index: ML_ANNOTATIONS_INDEX_ALIAS_READ,
+      size: 0,
+      body: {
+        query: {
+          bool: {
+            ...(earliestMs
```

Comment:
This should never be null.

Reply:
For usage with alerting - yes, `earliestMs` is always provided. But this service is re-usable, to potentially somebody will want to fetch annotations about delayed data without restricted time range.